vcloud.yaml

```diff
@@ -1,11 +1,334 @@
-openapi: 3.0.0
+openapi: 3.0.1
 info:
   title: V-Cloud API Reference
-  version: 1.0.0
+  description:
+    The V-Cloud API is a collection of web services used to interact with  Voci's
+    Automatic Speech Recognition (ASR) engine. V-Cloud is accessible to any programming
+    language capable of performing HTTP `POST` and `GET` requests. The service is
+    hosted at `https://vcloud.vocitec.com`.
+  version: 1.9.0-2022.06.02
+servers:
+  - url: https://vcloud.vocitec.com/
+tags:
+  - name: Transcribe Request
+    description:
+      The `/transcribe` endpoint accepts `POST` requests containing an audio
+      file, a zip file containing multiple audio files, or an audio file from an external
+      URL. Successful `/transcribe` requests return a `requestid` to retrieve the results.
+      If you specify a callback, results are sent to the specified callback address
+      as soon as the transcription process completes.
+  - name: Results Request
+    description:
+      The `/transcribe/result` endpoint accepts `GET` requests that contain
+      the `requestid` and `token` associated with the results being queried. Successful
+      `/transcribe/result` requests return a URL with the results as a zip or JSON file.
+  - name: Status Request
+    description:
+      The V-Cloud API is primarily used to transcribe audio and get the results,
+      but can also be used to determine if V-Cloud is available by querying the `/status`
+      endpoint.
 paths:
-  /:
+  /transcribe:
+    post:
+      tags:
+        - Transcribe Request
+      summary: Specify an audio file or URL source to transcribe
+      description:
+        Requests to the `/transcribe` endpoint must include the `file`
+        (or `url` for external audio sources) and `token` parameters. Additional [transcription
+        parameters](https://docs.vocitec.com/en/v-cloud-transcription-parameters.html)
+        are optional.
```

Comment:
Remove the `must include` portion of this.  Use the `required` in the schema to define requiredness.

Move the Additional Parameters into the schema as part of the `description` for any `additionalProperties` or whatnot that you set.